components/dialog/dialog.js

```diff
@@ -40,17 +58,19 @@ export class Dialog extends HTMLElement {
      */
     async connectedCallback() {
         this.shadowRoot.innerHTML = await fetch(import.meta.url.replace(".js", ".html")).then(response => response.text());
-        await this.#load();
+        await this.load();
```

Comment:
why public load?

Reply:
its because of this code in document-mock 

`globalThis.document.createElement = (tag, html) => {
    if (globalThis.__elementRegistry[tag] != null) {
        let result = mockElement(new globalThis.__elementRegistry[tag](), tag);

        if (result.load != null) {
            const load = result.load;
            result.load = async () => {
                createMockChildren(result);
                await load.call(result);
            }
        }

        return result;
    }

    return new ElementMock(tag);
}`